components/server/src/session-handler.ts

```diff
@@ -39,7 +42,31 @@ export class SessionHandlerProvider {
 
         options.store = this.createStore();
 
-        this.sessionHandler = session(options);
+        this.sessionHandler = (req, res, next) => {
+            let hasJWTCookie = false;
+
+            const cookies = parseCookieHeader(req.headers.cookie || "");
+            const jwtToken = cookies[SessionHandlerProvider.getJWTCookieName(this.config)];
+            if (jwtToken) {
+                // we handle the verification async, because we don't yet need to use it in the application
+                this.authJWT
```

Comment:
I get a tslint error here:

```
/workspace/gitpod/components/server/src/session-handler.ts:52:17
ERROR: 52:17  no-floating-promises  Promises must be handled appropriately
```

But honestly no clue why that is the case, handling looks bullet-proof... :shrug: 

I'd say let's slap a `/* tslint:disable-next-line */` on here.

Reply:
Added